db/mongo.go

```diff
@@ -16,12 +16,15 @@ import (
 // session must be registered first. A new copy of the master session can be
 // acquired. The session is provided through the DB type value.
 
+// ErrInvalidDBProvided is returned in the event that an uninitialized db is
+// used to perform actions against.
+var ErrInvalidDBProvided = errors.New("Invalid DB provided")
+
 //==============================================================================
 
 // mgoDB maintains a master session for a given database.
 type mgoDB struct {
-	dbName string
-	ses    *mgo.Session
+	ses *mgo.Session
```

Comment:
The only place we used it is where we were establishing a connection to the mgo db. As per the mgo docs: https://godoc.org/gopkg.in/mgo.v2#Session.DB if no db name is specified it will fallback to the db name provided in the dial options.